app/components/TextEditor/TextEditor.tsx

```diff
@@ -13,29 +14,41 @@ import Toolbar from './Toolbar';
 const TextEditor = (): React.ReactElement => {
   const { addNote } = useNoteStore();
 
+  // following these docs: https://tiptap.dev/docs/editor/extensions/functionality/placeholder
+  // but this doesn't appear to be working as expected - wonder why?
+  // TODO: check versions, check any next js configs that might be causing issues
   const editor = useEditor({
-    extensions: [StarterKit],
-    content: `
-      <p> Write your notes here! </p>`,
+    extensions: [
+      StarterKit,
+      Placeholder.configure({
+        placeholder: 'Write your notes here!',
+      }),
+    ],
+    content: '',
   });
```

Comment:
As noted, this is a test to get a nice little placeholder working, but I'm seeing some inconsistencies with the behavior I'm getting from TipTap and what's in the docs. 

I want to see how this performs in the amplify environment 🤷‍♀️